roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -311,6 +314,23 @@ listeners:
 #
 #allow_per_room_profiles: false
 
+# How long to keep redacted events in unredacted form in the database. After
+# this period redacted events get replaced with their redacted form in the DB.
+#
+# Defaults to `7d`. Set to `null` to disable.
+#
+#redaction_retention_period: 28d
+
+redaction_retention_period: {{ matrix_synapse_redaction_retention_period }}
+
+# How long to track users' last seen time and IPs in the database.
+#
+# Defaults to `28d`. Set to `null` to disable clearing out of old rows.
+#
+#user_ips_max_age: 14d
+
+user_ips_max_age: {{ matrix_synapse_user_ips_max_age }}
```

Comment:
These also need a CHANGELOG entry to say hey just so you know you are going to loose this stuff when you upgrade unless you set these to null.